pkg/common/utils/utils.go

```diff
@@ -223,3 +226,61 @@ func QueryVolumeDetailsUtil(ctx context.Context, m cnsvolume.Manager, volumeIds
 	}
 	return volumeDetailsMap, nil
 }
+
+// LogoutAllSessions will loggout all vCenter sessions and disconnect vCenter client
+func LogoutAllSessions(ctx context.Context, isMultiVCenterFssEnabled bool) error {
+	log := logger.GetLogger(ctx)
+	log.Info("logging out vCenter session")
+	virtualcentermanager := cnsvsphere.GetVirtualCenterManager(ctx)
+	vCenters := virtualcentermanager.GetAllVirtualCenters()
+	for _, vc := range vCenters {
+		if vc.Client == nil {
+			continue
+		}
+		log.Info("closing idle vCenter session")
+		vc.Client.CloseIdleConnections()
+		// logout vCenter session for list-view
+		mgr, err := cnsvolume.GetManager(ctx, vc, nil, false, isMultiVCenterFssEnabled, false, false)
```

Comment:
can we add following function in the manager interface

```
// GetAllManagerInstances returns all Manager instances
func GetAllManagerInstances(ctx context.Context) map[string]*defaultManager {
	if len(managerInstanceMap) != 0 {
		return managerInstanceMap
	}
	newManagerInstanceMap := make(map[string]*defaultManager)
	if managerInstance != nil {
		newManagerInstanceMap[managerInstance.virtualCenter.Config.Host] = managerInstance
	}
	return newManagerInstanceMap
}
```

Reply:
added this function to get manager instances